tsdb/head.go

```diff
@@ -2012,9 +2018,6 @@ func (s *memSeries) chunk(id int, chunkDiskMapper *chunks.ChunkDiskMapper) (chun
 	}
 	chk, err := chunkDiskMapper.Chunk(s.mmappedChunks[ix].ref)
 	if err != nil {
-		if err == chunks.ErrChunkDiskMapperClosed {
-			return nil, false
-		}
```

Comment:
Any reasons for removing this? It would create an unnecessary panic when we know we can get away with it safely (maybe a log with this?) :)